conf/application.conf

```diff
@@ -155,6 +156,7 @@ Prod {
         config {
             staticAssetUrlPrefix = "http://tax.service.gov.uk"
             staticAssetVersion = "/assets/2.230.0"
+            borderColour = "#005EA5"
```

Comment:
Should borderColour be rename to borderColor to be consistent? (i.e. assertRenderedBorderMatchesGivenColor(expectedColor) )